src/core/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ target_include_directories(
     Core
     PUBLIC
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
```

Comment:
Isn't this also required for the widgets library or do you set include paths manually in your project?

Reply:
I don't set it manually. So I suppose it gets inherited from Core as Widgets depends on it? I get linked to core automatically just by linking widgets, so for the include paths it should be exactly the same.

Reply:
Makes sense. Let me merge this now then and I can clean the rest of the includes later.